app/lib/actions/katello/repository/filtered_index_content.rb

```diff
@@ -17,7 +17,7 @@ def run
             repo.index_db_puppet_modules
           elsif repo.docker?
             # have to call Repository#index_db_docker_images to get the repo's tags
```

Comment:
@beav, You are correct that tags are now units in pulp; however, the need to perform the index remains the same.  The updated logic will retrieve the units (manifest & tag) and create them in katello, if needed.